aten/src/ATen/core/CachingHostAllocator.h

```diff
@@ -30,20 +30,17 @@ struct HostBlock {
   ska::flat_hash_set<S> streams_; // streams on which the block was used
 };
 
-/**
- * ComparatorSize is used for lookup support in the set of host memory blocks
- * using the block size.
- */
 template <typename B>
-struct ComparatorSize {
-  bool operator()(const B* a, const B* b) const {
-    if (a->size_ != b->size_) {
-      return a->size_ < b->size_;
-    }
-    return (uintptr_t)a->ptr_ < (uintptr_t)b->ptr_;
-  }
+struct alignas(64) FreeBlockList {
```

Comment:
Why the extra alignment constraint here?

Reply:
> Why the extra alignment constraint here?

This is to avoid cache line false sharing as we have a mutex in the free list struct. So, we want to give exclusive cache line to the mutex to avoid lots of cache line ping-ponging across CPU cores.

Reply:
Do we have a better value than the magic 64bytes being used here given the wide range of CPU pytorch runs on?